### PR TITLE
Add upcoming events.

### LINKED ISSUE #130: Add upcoming events
* `rstudio::conf(2019)`
* https://www.rstudio.com/workshops/extending-the-tidyverse/

### PR DESCRIPTION
**Pull request by batpigandme**
* Add rstudio::conf 2019.
* Add Building tidy tools.
* Fixes #130.

### CODE REVIEW

**Comment by hadley** [comment]
They don't seem to appear on the rendered site 😢

File: data/events.yaml [hunk 1]
```diff
@@ -22,6 +22,24 @@ events:
...
-    rstudio::conf 2018</a> covers all things RStudio, including workshops to
+    rstudio::conf 2018</a> covers all things RStudio, including workshops,
+    and talks to show you the latest and greatest features.
+
+- url: https://www.rstudio.com/workshops/extending-the-tidyverse/
+  title: Building tidy tools
+  where: San Franciso, CA
+  when:  May 17-18
+  iso8601: 2018-05-17
+  what: |
```
Comment (hadley) [inline data/events.yaml]: Also I think you swapped the what

**Comment by batpigandme** [comment]
I know!

File: data/events.yaml [hunk 2]
```diff
@@ -22,6 +22,24 @@ events:
   when:  Jan 31-Feb 2
   iso8601: 2018-01-31
   what: |
-    rstudio::conf 2018</a> covers all things RStudio, including workshops to
+    rstudio::conf 2018</a> covers all things RStudio, including workshops,
```
Comment (hadley) [inline data/events.yaml]: And can you remove the `</a>`?

File: data/events.yaml [hunk 3]
```diff
@@ -22,6 +22,24 @@ events:
...
+  where: San Franciso, CA
+  when:  May 17-18
+  iso8601: 2018-05-17
+  what: |
+    rstudio::conf 2019</a> covers all things RStudio, including workshops to
     teach you the tidyverse, and talks to show you the latest and greatest
     features.
+
+- url: https://www.rstudio.com/conference/
+  title: rstudio::conf 2019
```
Comment (hadley) [inline data/events.yaml]: Possibly these titles need to be wrapped in `"` because they contain `:`?

**Comment by hadley** [comment]
It's supposed to be by date 😬

**Comment by hadley** [comment]
But I have a vague recollection that turned out to be way harder than I thought. So just merge this.

**Pull request merged**